confd/templates/base-storm-topology/topology.properties.tmpl

```diff
@@ -88,7 +88,6 @@ discovery.db.write.repeats.time.frame = {{ getv "/kilda_discovery_db_write_repea
 # multi-FL
 floodlight.alive.timeout = {{ getv "/kilda_floodlight_alive_timeout" }}
 floodlight.alive.interval = {{ getv "/kilda_floodlight_alive_interval" }}
-message.blacklist.timeout = {{ getv "/kilda_message_blacklist_timeout" }}
```

Comment:
Please, remove it from topology.properties.example too.

Reply:
ok